currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -19,9 +19,11 @@ class ResCurrencyRateProviderECB(models.Model):
         selection_add=[('ECB', 'European Central Bank')],
     )
 
-    @api.model
-    def _get_supported_currencies(self, service):
-        if service == 'ECB':
+    @api.multi
+    def _get_supported_currencies(self):
+        self.ensure_one()
```

Comment:
No need to repeat the `self.ensure_one()`. Even more. This method doesn't need to ensure that.

Reply:
it's not repeating, super's `self.esnure_one()` may not be called

Reply:
OK, this is more a developer highlight than other thing, so it's not so important to be removed or not.

Reply:
But I see more cases. I have put this one, but I mean all the empty lines inside a method.

Reply:
Well, logical separation?

Reply:
We have talked about this other times:

* PEP8 encourages for not doing it.
* If you need a visual separation, better to put a comment line that also explains what the logical block is.
* This bloats the real number of lines/diff, and so on.

Reply:
But code like
```
a = something()
# here goes comment
b = something()
```
is less readable than
```
a = something()

# here goes comment
b = something()
```

If that's so strict, why it's not enforced via lint?